website/blog/flink-native-support/index.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Data lineage for Apache Flink
+date: 2024-11-15
+authors: [Leszczynski]
+description: Flink native lineage support will bring OpenLineage integration to the new era.
+---
+Apache Flink is a powerful stream processing engine widely adopted in the industry.
+Flink jobs are capable of emitting OpenLineage events for already two years.
+However, it is a recent joint effort of Flink and OpenLineage communities that will bring this integration into the new era.
+
+<!--truncate-->
+
+### Existing OpenLineage Flink integration
+
+As of November 2024, [OpenLineage Flink integration](https://github.com/OpenLineage/OpenLineage/tree/main/integration/flink) supports Flink versions `1.15` to `1.18` and emits OpenLineage events complying with the dataset naming model.
+It supports Kafka sources and sinks with Avro and Protobuf, JDBC and Cassandra connectors with extracting lineage from the queries executed.
+It also supports Hybrid sources. Overall, this is extremely helpful for the organizations to collect consistent cross-platform lineage graphs.
+
+The integration has been implemented with no changes to the Flink codebase which allowed OpenLineage community to boost its implementation in the early phase and recognize some pain points to be solved later.
+
+First, it is based on reflection mechanisms in Java and oftentimes extracts class members in an unsupported way.
+This makes the package difficult to maintain, causing from time to time classloading issues.
```

Comment:
either `causing, from time to time, classloading issues.` or `causing classloading issues from time to time.`